auto_install/install.sh

```diff
@@ -747,8 +747,8 @@ setCustomDomain() {
 confOpenVPN() {
     # Grab the existing Hostname
 	HOST_NAME=$(hostname -s)
-	# Generate a random, alphanumeric identifier of 16 characters for this server so that we can use verify-x509-name later that is unique for this server installation. Source: Earthgecko (https://gist.github.com/earthgecko/3089509)
-    NEW_UUID=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1)
+	# Generate a random UUID for this server so that we can use verify-x509-name later that is unique for this server installation.
+    NEW_UUID=$uuidgen -r
```

Comment:
@IcedComputer 
AFAIK this wrong syntax. It must be:
```
NEW_UUID=$(uuidgen -r)
```
But as my other comment, to avoid another dependency and external command, why not use:
```
NEW_UUID=$(</proc/sys/kernel/random/uuid)
```